requirements.txt

```diff
@@ -27,6 +28,7 @@ python-barbicanclient
 python-swiftclient>=3.2
 git+https://github.com/cloudbase/pywinrm.git@requests#egg=pywinrm
 PyYAML
+redis
```

Comment:
Although it's annoying that we have to specify these dependencies explicitly, I suggest we also add `python-mecached` as a dep so we can cover both most popular caching distros.
(The current Kolla we use install memcached, and only the updated Kolla @ionutbalutoiu is integrating will feature redis (and possible mecached too alongside it)

Reply:
It's already there:

https://github.com/cloudbase/coriolis/blob/master/requirements.txt#L25